packages/tezos-lwt-result-stdlib/tezos-lwt-result-stdlib.13.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: ["Tezos devteam"]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "2.9" }
+  "ocaml" { >= "4.12" }
+  "lwt" { >= "5.4.0" }
+  "alcotest-lwt" { with-test & >= "1.5.0" }
+  "qcheck-alcotest" { with-test & >= "0.18" }
+  "tezos-test-helpers" { with-test & = version }
+]
```

Comment:
```suggestion
    "ppx_inline_test" {with-test}
]
```

Reply:
# File "src/lib_time_measurement/ppx/test/dune", line 6, characters 18-33:
# 6 |  (preprocess (pps ppx_inline_test)))
#                       ^^^^^^^^^^^^^^^
# Error: Library "ppx_inline_test" not found.